### ISSUE TITLE
How to use redis connection string in <connectionStrings> while setting enableKeyspaceNotifications="true"

### ISSUE THREAD

**Issue by theperiscope** [MichaCo/CacheManager#254]
Hi @MichaCo ,

Is there a way to configure CacheManager to use a redis connection string in web.config's `<connectionStrings>` section, while enabling `enableKeyspaceNotifications`? I know there is a way to use `<connectionStrings>` but then I can't find a way to set `enableKeyspaceNotifications = true` if I did it.

I was thinking it would be helpful if the `<cacheManager>/<connections>/<connection>` tag  allowed for a `connectionStringName` attribute that points to a defined `<connectionStrings>` entry. As-is my redis connection strings are duplicated - one in the `<connectionStrings>` section for general redis operations, and one in the CacheManager... I would love to find a way to avoid it.

Thanks for the great work!

**Comment by MichaCo** [comment]
the keyspace notifications flag is specific to CacheManager and will not be recognized in the connection string parser. The `<connectionStrings>` section is a generic section and cannot be modified or anything either.

That being said, you could use CacheManager's configuration and use the static `RedisConfigurations` object to get a Redis configuration by name. The object returned has a `ConnectionString` property which can be passed in to a `StackExchange.Redis.ConnectionMultiplexer` if you want to use it outside of CacheManager.

**Comment by MichaCo** [comment]
Yeah, sounds like it would make kinda sense to reduce the redundancy of having the connection string twice. Although, does it really matter in production scenarios where this stuff is set in some dynamic/automated way anyways?

But considering that the old configuration system is pretty much dead in favor of the Extensions.Configuration stuff, I'm not planning to invest any time soon into adding things like that.
If you want to give it a try though, feel free to send a PR ;)

**Comment by MichaCo** [comment]
/closing - inactive - not going to fix
